styles/styles.css

```diff
@@ -0,0 +1,389 @@
+}
+
+header button {
+  display: flex;
+  width: 10em;
+  justify-content: center;
+  padding: .5em;
+  background-color: #4CAF50;
+  border: none;
+  color: white;
+}
+
+.checkout {
+  justify-content: center;
+  flex-direction: column;
+  align-self: flex-end;
+  display: flex;
+}
+
+i {
+  color: white;
+  font-size: 1em;
+  display: flex;
+  align-self: flex-end;
+  margin-bottom: 1em;
+}
+
+body {
+  padding-left: 5%;
+  padding-right: 5%;
+}
+
+nav {
+  background-color: #e1e7eb;
+  width: 100%;
+  padding: 1.5em;
+}
+
+nav li {
+display: inline;
+}
+
+nav a {
+ text-decoration: none;
+ font-weight: bold;
+ color: #18387d;
+ font-size: 1.2em;
+
+}
+h1 {
+  font-style: italic;
+  font-size: 1.5em;
+  color: #73c74a;
+}
+
+
+section p {
+  font-size: 1.5em;
+}
+
+
+.lead-article {
+  display: flex;
+  flex-direction: column;
+  align-self: flex-end;
+
+  margin-top: 0;
+
+}
+
+h2 a {
+  text-decoration: none;
+  color: #18387d;
+
+}
+
+h5 a {
+  text-decoration: none;
+  color: #29b1cc;
+  font-size: 1.2em;
+}
+
+aside h5 {
+  padding-bottom: 15%;
+  font-weight: bold;
+  color: #18387d;
+}
+
+h2 {
+  color: #18387d;
+  font-weight: bold;
+  font-size: 2em;
+  margin-bottom: 0;
+}
+
+h2, h3, h4, h5  {
+  font-weight: bold;
+  color: #18387d;
+}
+
+h4 {
+  font-size: 1.3em;
+}
+
+main article {
+  margin: 7%;
+  border: solid #e1e7eb .25em;
+  margin-left: 0;
+}
+
+
+main article img {
+  width: 100%;
+}
+
+
+ article button {
+   width: 90%;
+   padding: .5em;
+   display: flex;
+   justify-content: center;
+   align-self: center;
+   background-color: #4CAF50;
+   border: none;
+   color: white;
+   margin-bottom: 1em;
+   font-weight: bold;
+
+}
+
+main article p,h4,h5,button {
+  margin: .5em;
+
+}
+
+.all-button {
+  width: 70%;
+  text-align: center;
+  padding-top: .5em;
+  padding-bottom: 3em;
+}
+
+.all-button button {
+   width:25%;
+   background-color: #e1e7eb;
+   border: none;
+   color: #093c66;
+   font-size: 1em;
+   font-weight: bold;
+   padding: .75em;
+}
+
+footer {
+  background-color: #093c66;
+  text-align: center;
+  color: white;
+  width: 100%;
+  padding: 1em;
+
+}
+
+footer li {
+  display:inline;
+}
+
+footer a {
+  text-decoration: none;
+  color: white;
+  font-weight: bold;
+
+}
+
+@media (min-width: 320px) {
+  aside {
+    display: none;
+  }
+  section img {
+    max-width: 100%;
+    height: auto;
+  }
+  section {
+    width: 100%;
+  }
+}
+
+
+@media (min-width: 640px){
+  main {
+    display:grid;
+    grid-template-columns: repeat( 1, minmax(250px, 1fr) );
+  }
+
+  section img {
+    max-width: 100%;
+    height: auto;
+  }
+
+  section {
+    display: flex;
+    flex-direction: column;
+    margin-top: 5%;
+  }
+
+  aside {
+    display: none;
+  }
+
+}
+
+
+@media (min-width: 768px){
+  main {
+    display:grid;
+    grid-template-columns: repeat( 2, minmax(250px, 1fr) );
+ }
+
+}
+
+@media (min-width: 1024px){
+  main {
+    display:grid;
+    grid-template-columns: repeat( 2, minmax(250px, 1fr) );
```

Comment:
I like how you're adjusting this for each breakpoint. Nice use of media queries to handle this.